test/build.gradle.kts

```diff
@@ -48,6 +48,7 @@ kotlin {
                 // Our current compiler plugin tests only runs on JVM, so makes sense to keep them
                 // for now, but ideally they should go to the compiler plugin tests.
                 implementation("io.realm.kotlin:cinterop:${Realm.version}")
+                implementation("org.jetbrains.kotlinx:atomicfu:${Versions.atomicfu}")
```

Comment:
Does this work? I had problems using atomicfu in the test project

Reply:
This is required since I am asserting on the internal `realm.intermediateReferences.value`. So not applying it.